multihost_test/Bugzillas/test_subid_ranges_configurable.py

```diff
@@ -0,0 +1,303 @@
+""" Automation of Shadow-utils bugs """
+
+import pytest
+import subprocess
+import time
+import os
+from sssd.testlib.common.utils import SSHClient
+
+
+def execute_cmd(multihost, command):
+    """ execute cmd in client machine """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+class TestSubid(object):
+    """
+    This is for Shadow bugs automation
+    """
+    def test_subid_nochange_nsswitch(self, multihost,
+                                     create_backup,
+                                     create_localuser):
+        """
+        :Title: support pluggable data sources for
+        subid ranges configurable via /etc/nsswitch.conf
+        :id: 6ff93666-00c0-11ec-9be4-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1859252
+        :steps:
+            1. Define user range in /etc/subuid
+            2. Define group range in /etc/subgid
+            3. Create a process for user
+            4. Check newuidmap command is working
+            with no change in /etc/nsswitch.conf
+            5. Check newgidmap command is working
+            with no change in /etc/nsswitch.conf
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+            4. Should succeed
+            5. Should succeed
+        """
+        execute_cmd(multihost, "echo 'local_anuj:165536:65536' "
+                               "> /etc/subuid")
+        execute_cmd(multihost, "echo 'local_anuj:165536:65536' "
+                               "> /etc/subgid")
+        execute_cmd(multihost, "echo 'unshare -U bash' "
+                    "> /tmp/anuj.sh")
+        user = 'local_anuj'
+        ssh1 = SSHClient(multihost.client[0].sys_hostname,
+                         username=user, password='Secret123')
+        ssh2 = SSHClient(multihost.client[0].sys_hostname,
+                         username=user, password='Secret123')
+        (results1, results2, results3) = ssh1.exec_command('sh /tmp/anuj.sh')
+        find_id = "ps -ef | grep bash | grep local_anuj| tail -1"
```

Comment:
Please, use `user` variable instead of copy&pasting it's value.

Reply:
done